client/src/components/StockTicker/tickerBoard.css

```diff
@@ -0,0 +1,120 @@
+
```

Comment:
Like Stefan mentioned, I would probably just move this file to the `Styles` folder and `import` it from there.